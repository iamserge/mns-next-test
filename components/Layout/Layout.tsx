import Header from "../../components/Header/Header";
import { AuthProvider } from "../../providers/auth";

const Layout = ({children}) => {
    return (
        <>
            <AuthProvider>
                <Header />      
                {children}
            </AuthProvider>
        </>
    )
    
}

export default Layout;
