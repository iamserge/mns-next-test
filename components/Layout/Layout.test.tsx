import renderer from "react-test-renderer";
import Layout from "./Layout";

describe("Layout", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Layout><span>test</span></Layout>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
