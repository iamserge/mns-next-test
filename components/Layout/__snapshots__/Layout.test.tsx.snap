// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <div
    className="absolute w-full bg-white left-0 top-0 shadow"
  >
    <div
      className="flex justify-between py-6 px-6"
    >
      <h1
        className="text-2xl cursor-pointer"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        M&S TEST SHOP
      </h1>
      <div>
        <div>
          <input
            className="p-2 rounded border-2"
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="User id"
            type="text"
            value=""
          />
          <button
            className="hover:bg-gray-400 text-black text-base ml-5 p-2 px-4 rounded pointer-events-none bg-gray-200 "
            onClick={[Function]}
          >
            Log in
          </button>
        </div>
      </div>
    </div>
  </div>,
  <span>
    test
  </span>,
]
`;
